ga-ccd-definition/CaseEvent/Camunda/DashboardEventsGAspec-CUI-nonprod.json

```diff
@@ -0,0 +1,15 @@
+[
```

Comment:
The changes look good. Just a small suggestion, could we include 'WelshEnhancements' in the file name?